runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogData.java

```diff
@@ -19,7 +23,7 @@
     public static final LogData HOLE = new LogData(DataType.HOLE);
 
     @Getter
-    final DataType type;
+    DataType type;
```

Comment:
Yes, as a result of splitting the LogData constructor.  That split isn't needed anymore, so this & several other comments about LogData.java are addressed as I roll back the split.